collect/collect.py

```diff
@@ -870,6 +874,8 @@ def save_customizations_yaml(target):
     use the get_json_output() method to cache it.  If this ends up getting used elsewhere, should
     consider making the cache function output format independant.
 
+    If the santize_flag is set to True,
```

Comment:
`santize_flag` => `sanitize_flag`

Reply:
I like how I didn't actually finish that comment, too.